src/specs.js/App.test.js

```diff
@@ -0,0 +1,11 @@
+import { create } from 'react-test-renderer';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/megwali/spacex/pullRequest?prid=7693722)
```suggestion
import { create } from "react-test-renderer";
```